frontend/index.js

```diff
@@ -331,6 +333,10 @@ function clearOutput() {
 
 async function initUI() {
   document.addEventListener("keydown", ctrlEnterListener)
+  if (codeBackupDelayMillis >= 0) {
```

Comment:
Bit of a mysterious way to do feature toggling, I know - but was trying to avoid adding too many top level variables!